packages/engine/Source/Scene/GlobeSurfaceShaderSet.js

```diff
@@ -252,6 +257,9 @@ GlobeSurfaceShaderSet.prototype.getShaderProgram = function (options) {
       if (hasVertexNormals) {
         vs.defines.push("ENABLE_VERTEX_LIGHTING");
         fs.defines.push("ENABLE_VERTEX_LIGHTING");
+        fs.defines.push(
```

Comment:
Making this value a preprocessor `#define` instead of a `uniform` means that the shaders will require recompilation whenever this property is changed. Additionally, if a `uniform` is used, it will allow usage of a higher precision value than is provided here.

Reply:
Thanks for looking into this. I just pushed a change with your suggested change. I do agree it makes more sense